src/adapter/ads/nativeads.js

```diff
@@ -149,6 +151,7 @@ let NativeAdsAdapter = youbora.Adapter.extend({
 
   loadedAdListener: function (e) {
     this.adObject = e.payload.ad;
+    this.adPosition = e.payload.adType;
```

Comment:
`adType` is on the ad event just for backward compatibility.
The ad type should be taken from the ad break start event. If it doesn't exist on the ad break start event it's a bug in the player. is this the situation here?

Reply:
I think hat ad break start event is not even triggered in the environment you provided

Reply:
adbreakstart triggered. 
What environment you mean?

Reply:
This one
https://externaltests.dev.kaltura.com/player/library_Player_V3/smartPages/Player_V3_generic_page.php?cdnUrl=test&partnerId=1091&uiConfId=testCanary&selectedUiConf=15225670&entryId=empty&selectedEntryId=0_wifqaipd&compLst=&compVer=&compVers=&logs=deb&pllstByEntId=&pllstId=&cntDwnTTS=&cntDwnDur=&eng1=html&eng2=html&eng3=html&stPr1=&stPr2=&stPr3=&txtLang=default&audioLang=default&a_s=0&strPos=&adTag=noAd&adAftTime=&numRedirects=&dai=noDai&bumper=noBump&bumpPos=defBump&Youbora=on&WMPlace=

Reply:
this url doesnt contain the ad plugin. use this one
http://externaltests.dev.kaltura.com/player/library_Player_V3/smartPages/Player_V3_generic_page.php?cdnUrl=test&partnerId=1091&uiConfId=testCanary&entryId=empty&chgMed=empty&logs=deb&pllstByEntId=&pllstId=&cntDwnTTS=10&cntDwnDur=5&eng1=html&eng2=html&eng3=html&stPr1=def&stPr2=def&stPr3=def&txtLang=default&audioLang=default&a_s=0&strPos=0&plIn=on&adTag=vmPreMidPost&adAftTime=&Youbora=on&WMPlace=botRight

Reply:
True, sorry, that wasn't the link with ads. 
For the one you sent I don't see that event being triggered neither.

Reply:
for sure it triggered. but as i see in the your code `startBreakAdListener` doesn't bound to the event.
bind it to `AD_BFREAk_START` in `registerListeners`

Reply:
Updating